consensus/ibft/ibft.go

```diff
@@ -351,7 +351,7 @@ func (i *Ibft) buildBlock(snap *Snapshot, parent *types.Header) (*types.Block, e
 		Miner:      types.Address{},
 		Nonce:      types.Nonce{},
 		MixHash:    IstanbulDigest,
-		Difficulty: parent.Number + 1,   // we need to do this because blockchain needs difficulty to organize blocks and forks
+		Difficulty: 1,                   // we need to do this because blockchain needs difficulty to organize blocks and forks
```

Comment:
We should probably make this a global constant of the `ibft` package

Reply:
[Fixed](https://github.com/0xPolygon/polygon-sdk/pull/217/commits/cd26d0c65120515afd6c6e956c0e74a4542d01d9)